release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -284,6 +284,27 @@ For more information, see xref:../operators/operator_sdk/osdk-pkgman-to-bundle.a
 
 When running machines in vSphere, you can now set the `minReplicas` value to `0` in the `MachineAutoscaler` resource definition. When this value is set to `0`, the cluster autoscaler scales the machine set to and from zero depending on if the machines are in use. For more information, see the xref:../machine_management/applying-autoscaling.html#machine-autoscaler-cr_applying-autoscaling[MachineAutoscaler resource definition].
 
+[id="ocp-4-8-mco-upgrade-complete"]
```

Comment:
The mco block got updated but now i think it reads incorrect with the new first paragraph. In 4.8 if any pool is updating, we set Upgradeable = False